src/main/scala/ai/privado/cache/AuditCache.scala

```diff
@@ -82,7 +82,8 @@ class AuditCache {
     // TODO: This is using only JavaSemantic. We need to change to use based on language selection. I assume this can be handled inside Utilities.getEngineContext method itself
     val unfilteredPostSemanticsFlow = sinks
       .reachableByFlows(sources)(
-        Utilities.getEngineContext()(JavaSemanticGenerator.getSemantics(cpg, newPrivadoScanConfig, ruleCache))
+        Utilities
+          .getEngineContext(privadoScanConfig)(JavaSemanticGenerator.getSemantics(cpg, newPrivadoScanConfig, ruleCache))
```

Comment:
can we name it `privadoScanConfig` instead of new..?

Reply:
Addressed here: https://github.com/Privado-Inc/privado-core/pull/779/commits/4dc0d5ae6dba8ffc3b56b0dd6d716316799fa167